ComponentKit/Utilities/CKComponentDelegateAttribute.mm

```diff
@@ -30,16 +30,17 @@ CKComponentViewAttributeValue CKComponentDelegateAttribute(SEL selector,
     return {
       {
         std::string("Delegate-noop-") + sel_getName(selector) + "-",
-        ^(UIView *view, id value) {}, ^(UIView *view, id value) {}
+        ^(UIView *, CK::ViewAttribute::BoxedValue) {},
+        ^(UIView *, CK::ViewAttribute::BoxedValue) {}
       },
-      @YES  // Bogus value, we don't use it.
+      nil  // Bogus value, we don't use it.
     };
   }
 
   return {
     {
       std::string(sel_getName(selector)) + CKIdentifierFromDelegateForwarderSelectors(selectors),
-      ^(UIView *view, id value){
+      ^(UIView *view, CK::ViewAttribute::BoxedValue){
```

Comment:
Can we at least typedef the non-namespaced versions of this? CKViewAttributeBoxedValue. We're trying to avoid the CK:: usage since it looks weird in Obj-C